handler.hpp

```diff
@@ -3,6 +3,7 @@
 #define SABER_HANDLER_HPP
 
```

Comment:
Hmmm.. I wonder why this `handler.hpp` is still here?
I thought you `git mv`'d it to the `include/saber` directory?